planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_searcher.cpp

```diff
@@ -165,12 +177,18 @@ GoalCandidates GoalSearcher::search(const std::shared_ptr<const PlannerData> & p
       const auto transformed_vehicle_footprint =
         transformVector(vehicle_footprint_, autoware::universe_utils::pose2transform(search_pose));
 
-      if (isInAreas(transformed_vehicle_footprint, getNoParkingAreaPolygons(pull_over_lanes))) {
+      if (
+        parameters_.bus_stop_area.use_bus_stop_area &&
+        !isInAreas(transformed_vehicle_footprint, bus_stop_area_polygons)) {
+        break;
```

Comment:
```suggestion
        continue;
```

Reply:
sorry the name of `isInAreas` is bad, it return true, if it intersects.
for bus stop are the footprint should be completely in the area